google/cloud/spanner_admin_database_v1/types/backup.py

```diff
@@ -190,6 +190,7 @@ class State(proto.Enum):
         number=12,
         message=timestamp_pb2.Timestamp,
     )
+    database_dialect = proto.Field(proto.ENUM, number=10, enum=common.DatabaseDialect,)
```

Comment:
Using just `10` can be a bit confusing for new users. Probably should be done with a constant with a describing name.

Reply:
And I don't think `black` formatter will allow such a comma in the end

Reply:
Aren't these auto generated changes? Should the PR include these changes?